pkg/echo/echo.go

```diff
@@ -43,12 +44,20 @@ func echoHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// print the data sent to the echo server to the console
-	fmt.Printf("-- %s %s -> created %d\n", r.Method, r.URL.Path, http.StatusCreated)
+
+	// this should be in color1
+	color.Green("-- %s %s -> created %d\n", r.Method, r.URL.Path, http.StatusCreated)
+	// this can stay in default color
 	fmt.Printf("received %d readings:\n", len(payload.DataReadings))
-	for _, r := range payload.DataReadings {
-		fmt.Printf("%+v\n", r)
+	for i, r := range payload.DataReadings {
+		if i%2 == 0 {
+			color.Yellow("Reading:\n%s\n", prettyPrint(r))
+		} else {
+			color.Cyan("Reading:\n%s\n", prettyPrint(r))
+		}
```

Comment:
nit. I'd rework this a bit to avoid repetition and always print in the same line:

```
for i, r := range payload.DataReadings {
  c := color.New(color.FgYellow)
  if i%2==0 {
    c = color.New(color.Cyan)
  }
  color.New(c).Fprintf("#%s:\n%s\n", i, prettyPrint(r))
}
```